runtime/near-vm-runner/src/tests.rs

```diff
@@ -8,92 +8,29 @@ pub(crate) mod test_builder;
-                max_stack_height: config.limit_config.max_stack_height,
-                contract_prepare_version: match config.limit_config.contract_prepare_version {
-                    ExtContractPrepareVersion::V0 => ContractPrepareVersion::V0,
-                    ExtContractPrepareVersion::V1 => ContractPrepareVersion::V1,
-                    ExtContractPrepareVersion::V2 => ContractPrepareVersion::V2,
-                },
-                initial_memory_pages: config.limit_config.initial_memory_pages,
-                max_memory_pages: config.limit_config.max_memory_pages,
-                registers_memory_limit: config.limit_config.registers_memory_limit,
-                max_register_size: config.limit_config.max_register_size,
-                max_number_registers: config.limit_config.max_number_registers,
-                max_number_logs: config.limit_config.max_number_logs,
-                max_total_log_length: config.limit_config.max_total_log_length,
-                max_total_prepaid_gas: config.limit_config.max_total_prepaid_gas,
-                max_actions_per_receipt: config.limit_config.max_actions_per_receipt,
-                max_number_bytes_method_names: config.limit_config.max_number_bytes_method_names,
-                max_length_method_name: config.limit_config.max_length_method_name,
-                max_arguments_length: config.limit_config.max_arguments_length,
-                max_length_returned_data: config.limit_config.max_length_returned_data,
-                max_contract_size: config.limit_config.max_contract_size,
-                max_transaction_size: config.limit_config.max_transaction_size,
-                max_length_storage_key: config.limit_config.max_length_storage_key,
-                max_length_storage_value: config.limit_config.max_length_storage_value,
-                max_promises_per_function_call_action: config
-                    .limit_config
-                    .max_promises_per_function_call_action,
-                max_number_input_data_dependencies: config
-                    .limit_config
-                    .max_number_input_data_dependencies,
-                max_functions_number_per_contract: config
-                    .limit_config
-                    .max_functions_number_per_contract,
-                wasmer2_stack_limit: config.limit_config.wasmer2_stack_limit,
-                max_locals_per_contract: config.limit_config.max_locals_per_contract,
-                account_id_validity_rules_version: config
-                    .limit_config
-                    .account_id_validity_rules_version,
-            },
-        }
+pub(crate) fn test_vm_config() -> near_parameters::vm::Config {
+    let store = RuntimeConfigStore::test();
+    let config = store.get_config(PROTOCOL_VERSION).wasm_config.clone();
+    near_parameters::vm::Config {
+        vm_kind: config.vm_kind.replace_with_wasmtime_if_unsupported(),
```

Comment:
I’m curious, why did you need this? It seems to not have been needed before

Reply:
It (the call to `replace_with_wasmtime_if_unsupported`) is there in the previous version, though?

Reply:
Oh I had just seen the match and missed it just below, sorry 😅